lsp-ui-doc.el

```diff
@@ -823,7 +810,9 @@ It is supposed to be called from `lsp-ui--toggle'"
   :init-value nil
   :lighter ""
   :group lsp-ui-doc
-  :keymap `(([?q] . lsp-ui-doc-unfocus-frame)))
+  :keymap `(([?q] . lsp-ui-doc-unfocus-frame))
+  (when lsp-ui-doc-frame-mode
```

Comment:
If you're using my previous suggestion of advising `posframe--redirect-posfram-focus`, this block does needed